### PR TITLE
Add about page content

### LINKED ISSUE #17: About Page Content

### PR DESCRIPTION
**Pull request by Adrianv0512**
## Preview
image
image
image

## About Page Content
Added the content for the About Us page. This includes the structure of the page, all of the headings and text. Also added two new components which are used in the page: ValueCard and CardSlider. The value cards show the clubs values, and the card slider displays the eboard members. Only changed the about us, value card, and card slider css and js files. Also added all of the member photos into the public folder as well as the icons.

## Things to consider
- Add the rest of the members to the card slider
- Suggestions about the ordering of the section

### CODE REVIEW

**Review by boldenj2 — CHANGES_REQUESTED** [review]
Looks Good! Just need one minor change. On all of the e-board cards the text is a bit cluttered. Can we add some space between name, email, and bio on each card?

Example:
image

File: src/components/cardslider.js
Comment (scuruchima1) [inline src/components/cardslider.js]: Let's parameterize the CardSlider component to take in data of this type, and put this data list in the about us page component. This will ensure we can reuse the component in the future if need be and will make it easier for others to see where the data is coming from.

File: src/components/cardslider.js
Comment (scuruchima1) [inline src/components/cardslider.js]: Not sure if this is on purpose or if this is something for the future, but updating placeholder to the appropriate title could be good.

File: src/pages/aboutPage.js
Comment (scuruchima1) [inline src/pages/aboutPage.js]: A couple things we can improve on here:
- Since ValueCard is a react component we don't need a closing tag, instead we just need something like `<ValueCard url="" value="" text=""/>` and that will work as well.
- We should separate the parameters in each ValueCard declaration, so that each of the parameters are on different lines. This will make it easier to read what each value card holds.
```javascript
<ValueCard 
    url="" 
    value="" 
    text=""
/>
```

File: src/styles/aboutPage.css
Comment (scuruchima1) [inline src/styles/aboutPage.css]: Let's push these imports to the top of the file for consistency.

**Review by scuruchima1 — COMMENTED** [review]
Overall the code and design is good. I think some code formatting could help though and I have left comments for where it can improve. Also +1 Bolden's comments.

**Review by boldenj2 — APPROVED** [review]
Lgtm, make sure to resolve comments when you have addressed them. Also hold off on merging until @scuruchima1's approval.

**Comment by scuruchima1** [comment]
@boldenj2 @Adrianv0512 @AdalPadilla Will review soon just have to study for an exam. 😀

File: src/pages/aboutPage.js
Comment (boldenj2) [inline src/pages/aboutPage.js]: Put secretary before internal director and after treasurer.

**Review by boldenj2 — CHANGES_REQUESTED** [review]
One minor change. Please edit the order of the cards.

File: src/pages/aboutPage.js
Comment (scuruchima1) [inline src/pages/aboutPage.js:59]: I would put this data after the imports but before the AboutPage function declaration.

**Review by scuruchima1 — APPROVED** [review]
Everything looks good to me! Left a comment that is more so a nit about formatting. +1 to Bolden's requested changes.

**Review by boldenj2 — APPROVED** [review]

**Pull request merged**
